e2e/tests/functional/plugins/notebook/notebook.e2e.spec.js

```diff
@@ -209,13 +210,58 @@ test.describe('Notebook search tests', () => {
 
 test.describe('Notebook entry tests', () => {
     test.fixme('When a new entry is created, it should be focused', async ({ page }) => {});
-    test.fixme('When a telemetry object is dropped into a notebook, a new entry is created and it should be focused', async ({ page }) => {
-        // Drag and drop any telmetry object on 'drop object'
-        // new entry gets created with telemtry object
+    test('When an object is dropped into a notebook, a new entry is created and it should be focused', async ({ page }) => {
```

Comment:
Should all of these tests be marked as unstable? I'm not sure of the exact test process.